scripts/development/m3_stack/glide.lock

```diff
@@ -0,0 +1,178 @@
+hash: 60ff2b21e3037cb523207cdede4df79c59efc20e00b8c4d0825e18b61c121ced
```

Comment:
I think it must have been generated when I ran `make mock-gen-dbnode`. I'll remove, but should it be added to `.gitignore`?

Reply:
I wouldn't expect `make mock-gen-dbnode` to trigger this, not sure how it ended up here. I wouldn't want to add it to `.gitignore` specifically unless we knew how it got generated / if it would keep happening

Reply:
Hm true, I didn't look closely at the path. May have been leftover from previous work.